packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/ContentBlock.tsx

```diff
@@ -0,0 +1,24 @@
+﻿import { Text, RichText, Field, withDatasourceCheck } from '@sitecore-jss/sitecore-jss-nextjs';
```

Comment:
Looks like this and `lib/component-props` are just duplicates of the base nextjs template? Is there another basic component we can add here (hello world) instead of these files? Don't spend too much time as this is just a placeholder :)